pageserver/src/tenant/timeline.rs

```diff
@@ -2945,7 +3058,7 @@ impl Timeline {
 
             // Fsync all the layer files and directory using multiple threads to
             // minimize latency.
-            par_fsync::par_fsync(&layer_paths)?;
+            par_fsync::par_fsync(&layer_paths).context("failed to fsync all new layers")?;
```

Comment:
omit the `failed to`

Reply:
[06bf49c](https://github.com/neondatabase/neon/pull/3598/commits/06bf49cb9eafe38f2e95782cf7dcbc4dd28c52e1)